docs/bugs.json

```diff
@@ -1,4 +1,12 @@
 [
+    {
+        "name": "TupleAssignmentMultiStackSlotComponents",
+        "summary": "Tuple assignments with components that occupy several stack slots can result in invalid values.",
```

Comment:
Maybe give examples of such types?

Reply:
Added that now - I wasn't sure how far to extend the summary - so far it doesn't say that the number of components has to grow or shrink either, but if I add that as well, then the summary is already the full description...

Reply:
The summary should somehow reflect the "very low" severity. At the same time, I think we should not assume that readers of the summary know how many stack slots are occupied by each type.

Reply:
I agree on that - a bit tricky here, though, without getting too verbose - do you think it's fine with the examples of multi-stack-slot types or should I try to rephrase? Or should I use a complete list? The complete list is dynamic calldata array references, external function types and nested tuples, right?

Reply:
I think so.

Reply:
Updated the summary to the full list now.